hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestRollingUpgrade.java

```diff
@@ -83,37 +83,18 @@ public static void runCmd(DFSAdmin dfsadmin, boolean success,
   }
 
   @Rule
-  public TemporaryFolder folder = new TemporaryFolder();
-
-  /**
-   * Create a default HDFS configuration which has test-specific data directories.  This is
-   * intended to protect against interactions between test runs that might corrupt results.  Each
-   * test run's data is automatically cleaned-up by JUnit.
-   *
-   * @return a default configuration with test-specific data directories
-   */
-  public Configuration getHdfsConfiguration() throws IOException {
-    Configuration conf = new HdfsConfiguration();
```

Comment:
retain this, but just return the new config. allows for changes later

Reply:
Tests run fine without these configs. So, these configs don't seem to be actually needed. I am in favor to remove this function and directly use `new HdfsConfiguration();` to get a new conf object.